rest-api/alembic/versions/32df5fa890bf_raw_export_api.py

```diff
@@ -0,0 +1,299 @@
+   ps.sample_order_status_1sal,
+   ps.sample_order_status_1sal_time,
+   ps.sample_status_1sst8,
+   ps.sample_status_1sst8_time,
+   ps.sample_status_1pst8,
+   ps.sample_status_1pst8_time,
+   ps.sample_status_1hep4,
+   ps.sample_status_1hep4_time,
+   ps.sample_status_1ed04,
+   ps.sample_status_1ed04_time,
+   ps.sample_status_1ed10,
+   ps.sample_status_1ed10_time,
+   ps.sample_status_2ed10,
+   ps.sample_status_2ed10_time,
+   ps.sample_status_1ur10,
+   ps.sample_status_1ur10_time,
+   ps.sample_status_1sal,
+   ps.sample_status_1sal_time,
+   ps.num_completed_baseline_ppi_modules,
+   ps.num_completed_ppi_modules,
+   ps.num_baseline_samples_arrived,
+   ps.samples_to_isolate_dna,
+   ps.consent_for_cabor,
+   ps.consent_for_cabor_time,
+   (SELECT IFNULL(GROUP_CONCAT(
+      IF(ac.value = 'WhatRaceEthnicity_RaceEthnicityNoneOfThese',
+         'NoneOfThese',
+         TRIM(LEADING 'WhatRaceEthnicity_' FROM
+              TRIM(LEADING 'PMI_' FROM ac.value)))),
+    'None')
+    FROM questionnaire_response qr, questionnaire_response_answer qra,
+      questionnaire_question qq, code c, code ac
+    WHERE qra.end_time IS NULL AND
+          qr.questionnaire_response_id = qra.questionnaire_response_id AND
+          qra.question_id = qq.questionnaire_question_id AND
+          qq.code_id = c.code_id AND c.value = 'Race_WhatRaceEthnicity' AND
+          qr.participant_id = p.participant_id AND
+          qra.value_code_id = ac.code_id AND
+          ac.value != 'WhatRaceEthnicity_Hispanic'
+   ) race_codes,
+   (SELECT COUNT(ac.value)
+    FROM questionnaire_response qr, questionnaire_response_answer qra,
+      questionnaire_question qq, code c, code ac
+    WHERE qra.end_time IS NULL AND
+      qr.questionnaire_response_id = qra.questionnaire_response_id AND
+      qra.question_id = qq.questionnaire_question_id AND
+      qq.code_id = c.code_id AND c.value = 'Race_WhatRaceEthnicity' AND
+      qr.participant_id = p.participant_id AND
+      qra.value_code_id = ac.code_id AND
+      ac.value = 'WhatRaceEthnicity_Hispanic'
+   ) hispanic
+ FROM
+   participant p
+     LEFT OUTER JOIN hpo ON p.hpo_id = hpo.hpo_id
+     LEFT OUTER JOIN participant_summary ps ON p.participant_id = ps.participant_id
+     LEFT OUTER JOIN code state_code ON ps.state_id = state_code.code_id
+     LEFT OUTER JOIN code recontact_method_code ON ps.recontact_method_id = recontact_method_code.code_id
+     LEFT OUTER JOIN code language_code ON ps.language_id = language_code.code_id
+     LEFT OUTER JOIN code gender_code ON ps.gender_identity_id = gender_code.code_id
+     LEFT OUTER JOIN code sex_code ON ps.sex_id = sex_code.code_id
+     LEFT OUTER JOIN code sexual_orientation_code ON ps.sexual_orientation_id = sexual_orientation_code.code_id
+     LEFT OUTER JOIN code education_code ON ps.education_id = education_code.code_id
+     LEFT OUTER JOIN code income_code ON ps.income_id = income_code.code_id
+     WHERE p.withdrawal_status = 1 AND # NOT_WITHDRAWN
```

Comment:
So I think the only real difference between this and participant_view now is that we're filtering on withdrawal_status = 1 here.

That may be a good enough reason to have two views... but if the PPI QA people could handle having withdrawal status in the results and doing their own filtering, we wouldn't necessarily need this view. We should inquire.